### ISSUE TITLE
[Bug]: Unable to put initials only for identifiedBy when using DwC-A import

### ISSUE THREAD

**Issue by creplog** [SpeciesFileGroup/taxonworks#4063]
### Steps to reproduce the bug

```bash
1. Have .csv file containing Collection objects, including the field identifiedBy 
2. For a given collection object, populate identifiedBy with only person's initials (example: P., A.V.). The collection object's determiner only put their initials on the label (screenshot 1)
3. Use the DwC-A import workbench and upload .csv file 
4. import failed for collection object with initials only for identifiedBy. Status: Error last_name can't be blank (screenshot of bench with error included)
5. I tried to edit and rewrite the initials for the collection object within the created import workbench UI, record still errors
6. I can create a Person record with only initials (shown in last screenshot), but when I try to connect it to a record, it will not save (I saved the record after this screenshot, and then reloaded the page, and the determiner P., A.V. I added was not present)
...
```

### Screenshot

![image](https://github.com/user-attachments/assets/6baf2e43-f0ca-4f1b-a1f0-3ff95126e28b)
![image](https://github.com/user-attachments/assets/de377da2-43ad-4833-97f1-57863e838bb8)
![image](https://github.com/user-attachments/assets/3a541157-f412-49db-a1b7-43a377049a9d)
![image](https://github.com/user-attachments/assets/b3b8bf29-099f-4203-8304-50adb54b28e3)
![image](https://github.com/user-attachments/assets/826b6f23-8316-4a75-88ab-7f030575d682)
![image](https://github.com/user-attachments/assets/8df8f8d5-589c-4319-ba9e-2660d34c0542)

### Environment

Production

### Version

v0.44.0

### Browser Used

firefox

**Comment by LocoDelAssembly** [comment]
Looks the name is parsable as-is:
```ruby
3.3.4 :001 > DwcAgent.parse("P., A.V.")
 => [#<struct Namae::Name family="P.", given="A.V.", suffix=nil, particle=nil, dropping_particle=nil, nick=nil, appellation=nil, title=nil>] 
3.3.4 :002 > DwcAgent.parse("D., C.J.")
 => [#<struct Namae::Name family="D.", given="C.J.", suffix=nil, particle=nil, dropping_particle=nil, nick=nil, appellation=nil, title=nil>] 
3.3.4 :003 > 
```
Maybe `recordedBy` is the problem? Can you share the entire contents of the offending dataset row?

**Comment by LocoDelAssembly** [comment]
Completely unparsable text result is field be interpreted as blank. Not sure if we wan to change that at the expense of more frequent errored records?

I couldn't reproduce the problem with `P., A.V.` nor `D., C.J.`, it imports fine for me in local env.

**Comment by LocoDelAssembly** [comment]
Sorry, I can actually reproduce the problem! 

The importer after parsing the name it also cleans it with this third-party code: https://github.com/bionomia/dwc_agent/blob/6c87e49ff877afdf9fddffd21c0794e9acec719c/lib/dwc_agent/cleaner.rb#L27
```ruby

    # Cleans the passed-in namae object from the parse method and
    # re-organizes it to better match expected Darwin Core output.
    #
    # @param parsed_namae [Namae::Name] a Namae object
    # @return Namae::Name [Object] a new Namae object
```

I don't feel confident just removing this cleaner (@mjy?). The minimum requirement for parsed names is that the family name be complete, given names can be just initials.

**Comment by mjy** [comment]
@LocoDelAssembly Right, there is much more benefit to keep dwc_agent in the loop.  The real long-term solution is to include a `name` field for Person and use that when we can't reconstruct a parsing.

**Comment by dshorthouse** [comment]
Should it help, a newer version of `dwc_agent` has a utility method you might use to check if a parsed string once cleaned produces all nil attributes:
 
```
if cleaned_name != DwcAgent.default
   # do something otherwise store the unparsed input
end
```

**Comment by mjy** [comment]
There are many consequences of adding `name` to Person, it essentially sub-classes our understanding and adds a level of detection and parsing needs to the UI.  

* [ ] People with `name` but not parsed values will interfere with CSL rendering, we'll have to make this clear
* [ ] UI task that help to break down Name into components when possible

**Comment by dshorthouse** [comment]
You may not need any more rationale for a "name" attribute in your Person model, but here is an ORCID profile that was just recently added to Bionomia, https://orcid.org/0000-0002-5373-2585. It looks as if the various journals in which they and their colleagues have published their work have attempted various gymnastics to accommodate mononymous names.
